### PR TITLE
telfa + JWT

### PR DESCRIPTION
**Pull request by yep606**

### CODE REVIEW

File: src/main/resources/js/example.js [hunk 1]
```diff
@@ -6,35 +6,55 @@ const session = require('telegraf/session');
...
+
+    console.log(res.data.jwt)
+
+    let final = await axios({
+            method: 'get',
+            url: 'https://telfo.herokuapp.com/test',
+            headers: {
+                "Authentication": `Bearer ${res.data.jwt}`
+            }
+        }
```
Comment (mrgncvka) [inline src/main/resources/js/example.js:48]: Bearer context

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,11 +1,11 @@
-#spring.datasource.url= ${SPRING_DATASOURCE_URL}
-#spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
-#spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
-#spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url= ${SPRING_DATASOURCE_URL}
+spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
+spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+spring.jpa.hibernate.ddl-auto=update
```
Comment (mrgncvka) [inline src/main/resources/application.properties:4]: Create separated app.config files

**Review by mrgncvka — APPROVED** [review]
Check

File: src/main/java/base/controller/MainController.java [hunk 3]
```diff
@@ -23,16 +23,4 @@ public User add(@RequestBody User user){
         return userRepo.save(user);
     }
 
-    @GetMapping("/id/{id}")
-    public User getUser(@PathVariable Long id){
-        Optional<User> user = userRepo.findById(id);
-        return user.orElse(null);
-    }
-
-    @GetMapping("/hello")
```
Comment (mrgncvka) [inline src/main/java/base/controller/MainController.java:32]: Test #2

**Review by mrgncvka — CHANGES_REQUESTED** [review]

**Comment by yep606** [comment]
thx

**Comment by yep606** [comment]
thx

**Pull request merged**